src/app/shared/layout/header.component.ts

```diff
@@ -37,6 +44,38 @@ export class HeaderComponent implements OnInit {
     }
   }
 
+  viewBrowser() {
+    const routerUrl = this.router.url.substring(1); // remove the first '/' from string
+    const issueUrlIndex = routerUrl.indexOf('/'); // find the index of second '/'
+    let issueUrl: string;
+
+    // If can't find the index of second '/', then router is at the /issues (table list) page
+    if (issueUrlIndex < 0) {
+      // Apply filters to the issueUrl
+      issueUrl = '/issues'.concat(this.getSearchFilterString()).concat(this.getTeamFilterString());
+    } else {
+      // issueUrl will be from the second '/'
+      issueUrl = routerUrl.substring(issueUrlIndex);
+    }
+    shell.openExternal('https://github.com/'.concat(this.githubService.getRepoURL()).concat(issueUrl));
+  }
+
+  private getSearchFilterString() {
+    return this.FILTER_START.concat(this.issueService.getIssueSearchFilter());
+  }
+
+  private getTeamFilterString() {
+    if (this.issueService.getIssueTeamFilter() === 'All Teams') {
+      // Only exclude duplicates for phase 1 and 2
+      return (this.phaseService.currentPhase === 'phase3') ? '' : this.EXCLUDE_DUPLICATE;
```

Comment:
Note that although the filter is 'All Teams', it is still representing all the teams that the user is in. For example, if tutor is in team W12-3 and W12-4, we should filter by those them. 

Correctly me if I am wrong since I have not tested it, but from what I see, your meaning of 'All Teams' means no filter at all?

Reply:
Apologies, I thought that "All Teams" meant that the tutor can view every team in the module. I will change it to filter only the team that they are assigned to.

Reply:
Hi can I check that a tutor is assigned to every team under the same tutorial? (e.g W12 consist of team 3 and 4 so the tutor is assigned to both teams?)

Github label does not support OR operator so if I filter it by `+label:team.3+label:team.4` it will show "no result". The best I can do is to have a filter for `+label:tutorial.W12`

Reply:
> Hi can I check that a tutor is assigned to every team under the same tutorial? (e.g W12 consist of team 3 and 4 so the tutor is assigned to both teams?)

No, a tutorial can have 4 teams and 2 tutors (each tutor is allocated 2 teams).

Reply:
One way to do it is through the UI. We can have a dropdown menu instead of a button which will allow users to specify which team they want to be redirected to.

Reply:
Which phase are we talking about here? In the _tutor-moderation_ phase, the script assigns the issue to the respective tutor.

Reply:
The "Dev-Response" phase (phase 2)

Reply:
> One way to do it is through the UI. We can have a dropdown menu instead of a button which will allow users to specify which team they want to be redirected to.

There is already a dropdown for the tutor to select which team's issues they want to view

Reply:
> There is already a dropdown for the tutor to select which team's issues they want to view

I meant for the "View in Github" button.

Reply:
> The "Dev-Response" phase (phase 2)

But this is also applicable to tutor-moderation phase right? Every phase has that "View in Github" button.

Reply:
Yup, actually it is also applicable for that phase

Reply:
Should I go ahead and add a dropdown for the "Open in Browser" button?

Reply:
Sure, I am just suggesting, don't really need to follow it. Prof @damith, what do you think about the change in UI?

Reply:
@KevinCJH ask me tomorrow when I drop by.

Reply:
I have changed the code to hide the "View in Browser" button when viewing "All Teams"